signing/src/main/java/tech/pegasys/web3signer/signing/metrics/SigningMetricCategory.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 ConsenSys AG.
```

Comment:
2022?

Reply:
I think copy and pasted from the Signing protection metric category and forgot to change date. Will fix it up.

Reply:
done

Reply:
done